e2e/packages/sync-test/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "test": "vitest --run",
+    "create:db": "psql postgres://127.0.0.1/postgres -c \"CREATE DATABASE postgres_e2e;\"",
+    "test": "DATABASE_URL=postgres://127.0.0.1/postgres_e2e vitest --run",
     "test:ci": "pnpm run test"
```

Comment:
I assume the `test` script will overwrite the env var passed in by CI itself? we might have to change this to just be `vitest --run` or make a separate `test:local` script that passes in the `DATABASE_URL`

Reply:
ahh you're right, didn't realize `test:ci` just calls `test`